src/transaction/log_replication_atomic.cpp

```diff
@@ -20,8 +20,9 @@
 #include "log_replication_atomic.hpp"
 #include "log_replication_jobs.hpp"
 
-#include "log_recovery_redo_parallel.hpp"
 #include "heap_file.h"
+#include "log_recovery_redo_parallel.hpp"
```

Comment:
why does it come down?

Reply:
Due to sorting alphabetically like the other files, it has been modified

Reply:
Then, doesn't we need to go up the `heap_file.h`? There are other headers starting with `L`.

Reply:
It appears that the `log_replication_*` header files are kept separately, while the rest of the header files are declared below. Therefore, I have sorted the lower section alphabetically without altering the paragraph above.

In `log_checkpoint_info.cpp` and other places, it seems that directly related header files are declared alphabetically first, and the rest of the header files are declared alphabetically in the following section.